R/autoplot.summary.seroincidence.by.R

```diff
@@ -15,20 +15,22 @@
 #' library(ggplot2)
 #'
 #'
-#' xs_data = load_pop_data("https://osf.io/download//n6cp3/") %>%
+#' xs_data <- load_pop_data("https://osf.io/download//n6cp3/") %>%
 #'   clean_pop_data()
-#' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
-#' filter(antigen_iso == c("HlyE_IgA", "HlyE_IgG")) %>%
-#'   slice(1:100, .by=antigen_iso) #reduce dataset for the purposes of this example
-#' noise = load_noise_params("https://osf.io/download//hqy4v/")
 #'
-#' est2 = est.incidence.by(
-#' strata = c("catchment"),
-#' pop_data = xs_data %>% filter(Country == "Pakistan"),
-#' curve_params = curve,
-#' noise_params = noise %>% filter(Country == "Pakistan"),
-#' antigen_isos = c("HlyE_IgG", "HlyE_IgA"),
-#' #num_cores = 8 #Allow for parallel processing to decrease run time
+#' curve <- load_curve_params("https://osf.io/download/rtw5k/") %>%
+#' filter(antigen_iso %in% c("HlyE_IgA", "HlyE_IgG")) %>%
```

Comment:
are you doing the indenting by hand, or did you find a way to automate it? Either way, line 22 needs an indent.

Reply:
Not fancy enough to automate it, so I did it by hand haha.